imputation/ts_impute_test.go

```diff
@@ -0,0 +1,112 @@
+package imputation
+
+import (
+	"testing"
+)
+
+func computeMean(s TimeSeries) float64 {
+	var mean float64
+	for _, v := range s {
+		mean += v
+	}
+	mean = mean / float64(len(s))
+	return mean
+}
+
+func createOutput(ts TimeSeries, desiredValues []string) TimeSeries {
+	out := make(TimeSeries, len(ts)+len(desiredValues))
+	for k, v := range ts {
+		out[k] = v
+	}
+	m := computeMean(ts)
+
+	for _, k := range desiredValues {
+		out[k] = m
+	}
```

Comment:
```suggestion
        if len(desiredValues) > 0 {
	  m := computeMean(ts)
	  for _, k := range desiredValues {
		out[k] = m
	  }
	}
```

This avoids division by zero in the mean function.

Reply:
Do you mean if len(ts) > 0?